src/Editor/main.cpp

```diff
@@ -24,9 +24,12 @@
 
 int main() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Return value of function freopen() is not used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AignoredReturnValue)